app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -68,6 +71,42 @@ protected function create(array $data)
             'last_name' => $data['last_name'],
             'email' => $data['email'],
             'password' => bcrypt($data['password']),
+            'email_token' => str_random(32)
         ]);
     }
+
+    /**
+     * Handle a registration request for the application.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function register(Request $request)
+    {
+        $this->validator($request->all())->validate();
+        event(new Registered($user = $this->create($request->all())));
+        dispatch(new SendVerificationEmail($user));
+        return view('email.verification');
+    }
+
+    /**
+     * Handle a registration request for the application.
```

Comment:
Just noticed you made a little copy paste mistake here. And no type hint for $token.